code/modules/power/supermatter/supermatter_delamination/_sm_delam.dm

```diff
@@ -74,17 +74,46 @@ GLOBAL_LIST_INIT(sm_delam_list, list(
 	return
 
 /// Added to an examine return value.
-/// [/obj/machinery/power/supermatter_crystal/proc/examine]
+/// [/obj/machinery/power/supermatter_crystal/examine]
 /datum/sm_delam/proc/examine(obj/machinery/power/supermatter_crystal/sm)
 	return list()
 
 /// Add whatever overlay to the sm.
-/// [/obj/machinery/power/supermatter_crystal/proc/overlays]
+/// [/obj/machinery/power/supermatter_crystal/update_overlays]
 /datum/sm_delam/proc/overlays(obj/machinery/power/supermatter_crystal/sm)
 	if(sm.final_countdown)
-		return list(mutable_appearance(sm.icon, "causality_field"))
+		return list(mutable_appearance(icon = sm.icon, icon_state = "causality_field", layer = FLOAT_LAYER))
```

Comment:
why manually set the icon and state args?

Reply:
Not sure i understand what you mean here. You mean why i made them named args?

Reply:
yeah

Reply:
Just thought its more readable to be honest, no real technical reason.